Azure/communication-ui-library-android
# [BugFix]  Sort participant list by display name

## Purpose
To show the participant list alphabetically in the call. Also handle the suffix of local participant and empty name.
impact: participant list

## Does this introduce a breaking change?
```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Join a group call with multiple participants' with various of display names to see if the participant list showing up alphabetically.
